public/src/client/topic/posts.js

```diff
@@ -33,10 +33,12 @@ define('forum/topic/posts', [
 		ajaxify.data.postcount ++;
 		postTools.updatePostCount(ajaxify.data.postcount);
 
-		if (config.usePagination) {
-			onNewPostPagination(data);
-		} else {
-			onNewPostInfiniteScroll(data);
+		if (config.scrollToMyPost) {
```

Comment:
Can you change this to `ajaxify.data.scrollToMyPost` and add `scrollToMyPost` here https://github.com/NodeBB/NodeBB/blob/master/src/controllers/topics.js#L261 so it is passed to the client.